### ISSUE TITLE
Add mod to modrinth please

### ISSUE THREAD

**Issue by FaNt4zMa** [nanite/TheObsidianBoat#25]
Hey there! Could you pretty please upload this mod to Modrinth? I cannot find any other fireproof boat mod available for Fabric there and I'd like to use this in my modpack. Or can you give me permission to use it anyway? Thanks!

**Comment by Alter73** [comment]
Yes, I agree. It would be great if this mod was also available in Modrinth. This is especially useful when creating modpacks.
